src/v3/test/e2e/.testcaferc.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   },
   appCommand: 'test/e2e/start-app-e2e.sh',
   browsers: 'chrome:headless',
-  concurrency: 3,
+  concurrency: 1,
```

Comment:
new version of testcafe does not like when concurrency is higher than the amount of tests